website/src/app/kb/deploy/resources/readme.mdx

```diff
@@ -46,6 +46,24 @@ From there, you can select the type of Resource you want to create:
   Resource.
 </Alert>
 
+#### Routing order for overlapping addresses
+
+When multiple Resources' addresses overlap, the Resource with the more specific
+address will be used.
+
+For CIDR Resources, this means the Resource with the longest prefix length will
```

Comment:
```suggestion
For CIDR Resources, an address with a longer prefix is more specific than a shorter one. For example: `10.0.0.0/16` is more specific than `10.0.0.0/8`. IP Resources are essentially addresses with `/32` prefix and thus always more specific than any other CIDR.
```

This would avoid some word repetition.